setup_wizard/parsers/material_data_json_parsers.py

```diff
@@ -37,6 +37,29 @@ def __get_rgba_colors(self, material_json_value):
         return (r, g, b, a)
 
 
+class UnknownHoyoStudioMaterialDataJsonParser(HoyoStudioMaterialDataJsonParser):
```

Comment:
We determined this was a specific version and that it was reverted later on.

This is still included in the event that this Parser is needed.